.travis.yml

```diff
@@ -65,7 +65,7 @@ script:
   - cd $(python -c 'import theano; import os; print(os.path.split(theano.__file__)[0])')
   - echo "$PART"
   - cd -; cd Theano
-  - python -c 'import theano; print(theano.config)'
+  - python -c 'import theano; print theano.config.__str__(print_doc=False)'
```

Comment:
This will break on python 3 because you removed the parentheses around the argument to `print`